src/scene/lightmapper.js

```diff
@@ -297,7 +297,7 @@ class Lightmapper {
         }
 
         // mesh instances from render component
-        if (node.render && node.render.enabled) {
+        if (node.render?.enabled) {
```

Comment:
Have "optional chaining" been investigated in terms of Babel. How does it compile into ES5. If it is good, please consider PR to update [CONTRIBUTING.md](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md).

Reply:
this line
`if (node.render?.enabled)`

compiles to this
`if ((_node$render = node.render) != null && _node$render.enabled)`

Reply:
And if you do 3-5 chains in single line, does it look ok?

Reply:
Slightly longer example. The es5 is not the easiest to read unfortunately. 
From
```
        var graph;
        if (this.entity.model?.model?.graph) {
            graph = this.entity.model?.model?.graph;
        } else {
            // animating hierarchy without model
            graph = this.entity;
        }
```
to
```
var graph;

if ((_this$entity$model = this.entity.model) != null && (_this$entity$model$mo = _this$entity$model.model) != null && _this$entity$model$mo.graph) {
	var _this$entity$model2, _this$entity$model2$m;

	graph = (_this$entity$model2 = this.entity.model) == null ? void 0 : (_this$entity$model2$m = _this$entity$model2.model) == null ? void 0 : _this$entity$model2$m.graph;
} else {
	graph = this.entity;
}
```

Reply:
Readability is not great indeed, but ES5 not meant to be read (unless debugging babel specific issues, which we haven't met yet). So probably is fine. Worth adding it to CONTRIBUTING.md.

Reply:
Done: https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md#whitelisted-es6-features